tests/test_transformDiaSourceCatalog.py

```diff
@@ -175,6 +175,7 @@ def test_computeBBoxSize(self):
     def test_bit_unpacker(self):
```

Comment:
Put a TODO with a link to the ticket for the removal of the config, or just remove the test entirely if you've totally removed the functionality (which I think we probably should do on this ticket anyway).